core/src/main/scala/runtime/endpoints.scala

```diff
@@ -37,6 +40,14 @@ object implicits {
   implicit def endpointsHandler[M[_]: Applicative]: EndpointsM.Handler[M] =
     new EndpointsM.Handler[M] {
 
+      def protoMetric: M[HttpService] =
+        HttpService {
+          case request @ GET -> Root / "proto" / "metric" =>
+            StaticFile
+              .fromFile(new File("core/src/main/protobuf/metric.proto"), Some(request))
```

Comment:
I wouldn't expose an `IO` failure in the api layer, (smells to 500 error :P )

Reply:
I don't understand this comment. I am putting an getOrElseF after `fromFile`, so if we don't find the file, we will return a 404 not found and it won't smell to 500 error